open-hackathon-server/src/hackathon/config_sample.py

```diff
@@ -92,6 +92,7 @@
         "username": "",
         "password": "",
         "email_templates": {
+            "path": "/home/opentech/open-hackathon/open-hackathon-server/src/hackathon/resources/email/",
```

Comment:
see https://github.com/msopentechcn/open-hackathon/blob/master/open-hackathon-server/src/hackathon/storage/local_storage.py#L120 for example